api-reference/10 UI Components/dxScheduler/1 Configuration/onAppointmentTooltipShowing.md

```diff
@@ -5,28 +5,27 @@ default: null
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Occures before showing an appointment's tooltip.
 
 ##### param(e): Object
-<!-- Description goes here -->
+Information about the event.
 
 ##### field(e.appointments): Array<Object>
-<!-- Description goes here -->
+An array of appointments. 
 
 ##### field(e.cancel): Boolean
-<!-- Description goes here -->
+Allows you to prevent a tooltip showing.
 
 ##### field(e.component): dxScheduler
-<!-- Description goes here -->
+The UI component's instance.
 
 ##### field(e.element): DxElement
-<!-- Description goes here -->
+#include common-ref-elementparam with { element: "UI component" }
 
 ##### field(e.model): any
 The model data. Available only if you use Knockout.
 
 ##### field(e.targetElement): DxElement
-<!-- Description goes here -->
+An element to which target the appointment.
```

Comment:
Do you mean the following?

`The target element of the appointment.`

If not, please clarify.

Reply:
I agree with you. Thank you.

However I've noticed my error. I mean tooltip instead of appointment.

Updated to `The target element of the tooltip.`